weave/trace_server/external_to_internal_trace_server_adapter.py

```diff
@@ -314,3 +314,22 @@ def feedback_query(self, req: tsi.FeedbackQueryReq) -> tsi.FeedbackQueryRes:
     def feedback_purge(self, req: tsi.FeedbackPurgeReq) -> tsi.FeedbackPurgeRes:
         req.project_id = self._idc.ext_to_int_project_id(req.project_id)
         return self._ref_apply(self._internal_trace_server.feedback_purge, req)
+
+    def cost_create(self, req: tsi.CostCreateReq) -> tsi.CostCreateRes:
+        req.project_id = self._idc.ext_to_int_project_id(req.project_id)
+        return self._ref_apply(self._internal_trace_server.cost_create, req)
+
+    def cost_purge(self, req: tsi.CostPurgeReq) -> tsi.CostPurgeRes:
+        req.project_id = self._idc.ext_to_int_project_id(req.project_id)
+        return self._ref_apply(self._internal_trace_server.cost_purge, req)
+
+    def cost_query(self, req: tsi.CostQueryReq) -> tsi.CostQueryRes:
+        original_project_id = req.project_id
+        req.project_id = self._idc.ext_to_int_project_id(original_project_id)
+        res = self._ref_apply(self._internal_trace_server.cost_query, req)
+        for cost in res.results:
+            if "pricing_level_id" in cost:
```

Comment:
this will have to be changed if we allow different scope costs? When is this not the project_id? When do we expect this error to happen? what has to break upstream?

Reply:
This check will have to be extended when org costs are implemented, Ill add a comment.
Im not sure when this check would be triggered. We do have the check for both the feedback and calls query